hepcrawl/spiders/elsevier_spider.py

```diff
@@ -554,20 +537,28 @@ def _get_ref_years(self, ref):
     def _parse_references(self, ref, label):
         """Parse all the references."""
         reference = {}
-        textref = ref.xpath(".//ce:textref//text()").extract()
+
+        def add_items_to_reference(**kwargs):
```

Comment:
well, just returning the initialized reference will allow you to run the same function on any part of the code, something like:

```python
def add_items_to_references(reference=None, **kwargs):
    reference = {} if reference is None else reference
    for field, value in kwargs.iteritems():
        if value:
           reference[field] = value

        return reference
```

that way you can reuse it with an existing reference or just call it to create one (that's not so nice actually... as the name does not reflect that behavior very well though), like this:

```python
myref = {'something': 234}
myref = add_items_to_reference(myref, item1=123, item2='something else',...)

# or directly with
myref = add_items_to_reference(
    something=234,
    item1=123,
    item2='something else'
)
```
for the other items, well, it gets a bit more complicated, if you want to still use that function you can try adding an extra parameter, something like 'check_func' where you pass a lambda for the test to run... if it gets more complicated, then it's not worth using that function for that key.